assets/script.js

```diff
@@ -0,0 +1,75 @@
+function main() {
+  // correctAnswer and currentAnswer used for game logic
+  const correctAnswer = ["button1", "button3", "button2"];
+  let currentAnswer = [];
+  const gameOutcomeEl = document.getElementById("gameOutcome");
```

Comment:
its minor, but I like you labeled your elements with El.  Its clean, and caught my eye, and I instantly knew what I was looking at.